common/lib/xmodule/xmodule/js/src/discussion/display.coffee

```diff
@@ -1,4 +1,4 @@
 class @InlineDiscussion extends XModule.Descriptor
   constructor: (element) ->
     @el = $(element).find('.discussion-module')
-    @view = new DiscussionModuleView(el: @el)
+    @view = new DiscussionInlineView(el: @el)
```

Comment:
Nit: I was thinking about the name for this, and it isn't really just an inline view. Maybe we should call it something like ``DiscussionTopicView``, because it can be used to show all the contents of a discussion topic. It seem a little odd to have the teams view use a component called ``DiscussionInlineView`` because it isn't an inline discussion in the sense we use it, i.e. inline in a course.